ops/mappings.json

```diff
@@ -8,6 +8,13 @@
         "mapping": "DAOToken",
         "arcVersion": "0.0.1-rc.16"
       },
+      {
+        "name": "NewSignalScheme",
+        "dao": "address",
+        "mapping": "SignalScheme",
```

Comment:
I thought we are using GenericScheme ?
is that needed ?

Reply:
@cryptogoth ?

Reply:
Hi @orenyodfat I've never seen that scheme before, and haven't delved into daostack repo much :)

It sounds cool though.